PowerShell/Deploy/BuildNuspecFromPsd1.ps1

```diff
@@ -2,23 +2,22 @@
 param (
     [Parameter()]
     [string]
-    $RequiredModulesRepo
+    $RequiredModulesRepo = 'PSGallery'
 )
 
 . "$PSScriptRoot/Get-Config.ps1" -RequiredModulesRepo:($RequiredModulesRepo)
 # $nuspecFiles = @{ src = 'en-Us/**;Private/**;Public/**;JumpCloud.psd1;JumpCloud.psm1;LICENSE'; }
 $nuspecFiles = @(
-    @{src = "en-Us/**/*.*"; target = "en-Us" },
```

Comment:
We had to change this to prevent new directories from showing up in the module folder
![Screen Shot 2022-08-04 at 10 52 48 AM](https://user-images.githubusercontent.com/54448601/182907069-7d2e78b6-d72a-4dcd-8771-71ca97106247.png)

Now the directory structure is correct :)

![Screen Shot 2022-08-04 at 10 53 15 AM](https://user-images.githubusercontent.com/54448601/182907147-ac8a83a2-a476-4574-b9d2-d1430d22a9b6.png)